pubpub/platform
# Add pubId to form memberships

## Issue(s) Resolved
Closes https://github.com/pubpub/platform/issues/842

## High-level Explanation of PR
Prevent users invited to edit a pub from editing other pubs (previously could do this by swapping out the `pubId` parameter. This adds a `pubId` to the form memberships table so that invited users are scoped to a pubId.

## Test Plan
* Go to a pub in CrocCroc in the Submitted stage and run the action "Send Review email"
* Send the email to a member who is only a contributor (i.e. `none@pubpub.org`)
* Visit inbucket http://localhost:54324/monitor and find the email that was just sent
* View the email in 'plain text' form and copy the link 
* Open the link in an incognito tab (or any browser that isn't signed in)
* You should see the form
* Now copy the link, but swap out the `pubId` value. i.e. swap out the `[pubId]` portion here `http://localhost:3000/magic-link?token=[token]&redirectTo=%2Fc%2Fcroccroc%2Fpublic%2Fforms%2Freview%2Ffill%3FpubId%3D[pubId]`
* Paste that link into another place you aren't signed in and you should see a 404 page
* Now instead of replacing with another `pubId`, just remove the pubId, i.e. `http://localhost:3000/magic-link?token=[token]&redirectTo=%2Fc%2Fcroccroc%2Fpublic%2Fforms%2Freview%2Ffill%3F` 
* You should be able to access the create page and be able to create a pub still

## Screenshots (if applicable)

## Notes
